### PR TITLE
City template

### PR DESCRIPTION
**Pull request by ssynowiec**

### CODE REVIEW

File: src/app/(cities)/wroclaw/page.mdx [hunk 1]
```diff
@@ -37,4 +42,49 @@ export const metadata = {
...
 };
 
-# Wrocław
\ No newline at end of file
+<CityBanner city='Wrocław' background='wroclaw.jpeg' />
+
+<LocalGroups
+	localGroups={['https://www.meetup.com/meet-js-wroclaw/', 'https://www.facebook.com/groups/326774397348491', 'https://lu.ma/user/usr-eNzoaGXYjizOBQI']} />
+
+<FAQ questions={[{ id: '1', question: 'How to become Wrocław meetup sponsor?', answer: 'contact' }, {
```
Comment (dzienisz) [inline src/app/(cities)/wroclaw/page.mdx]: Should we save it in a different file? I assume we won't create any back-office soon or anytime, so it's okay, but I would put it into a separate file.

**Review by dzienisz — APPROVED** [review]

File: src/app/(cities)/wroclaw/page.mdx (hunk 1 above)
Comment (ssynowiec) [reply src/app/(cities)/wroclaw/page.mdx]: Every city in different file?

Comment (dzienisz) [reply src/app/(cities)/wroclaw/page.mdx]: data for FAQ in separate file

**Comment by ssynowiec** [comment]
@dzienisz is this what you think? https://github.com/ssynowiec/meetjs.pl/pull/31/commits/a6e57de838d846667044e78f07230097936f1b12

File: src/app/(cities)/wroclaw/page.mdx (hunk 1 above)
Comment (dzienisz) [reply src/app/(cities)/wroclaw/page.mdx]: Overall we can merge it. It's not a blocker, just a suggestion.

**Comment by dzienisz** [comment]
> @dzienisz is this what you think? [a6e57de](https://github.com/ssynowiec/meetjs.pl/commit/a6e57de838d846667044e78f07230097936f1b12)

Great!

**Pull request merged**